pkg/scripts/render.go

```diff
@@ -58,19 +79,24 @@ var (
 				{{- if .FORCE }}
 				--allow-downgrades \
 				{{- end }}
-				docker-ce=5:{{ $DOCKER_VERSION_TO_INSTALL }} docker-ce-cli=5:{{ $DOCKER_VERSION_TO_INSTALL }}
-			sudo apt-mark hold docker-ce docker-ce-cli
+				docker-ce=5:{{ $DOCKER_VERSION_TO_INSTALL }} \
+				docker-ce-cli=5:{{ $DOCKER_VERSION_TO_INSTALL }} \
+				containerd.io=%s
+			sudo apt-mark hold docker-ce docker-ce-cli containerd.io
+
 			sudo systemctl daemon-reload
+			sudo systemctl enable --now containerd
 			sudo systemctl enable --now docker
 			`,
 			defaultDockerVersion,
 			defaultLegacyDockerVersion,
 			latestDockerVersion,
+			defaultContainerdVersion,
 		),
 
 		"yum-docker-ce-amzn": heredoc.Docf(`
 			{{- if or .FORCE .UPGRADE }}
-			sudo yum versionlock delete docker cri-tools || true
+			sudo yum versionlock delete docker cri-tools containerd
```

Comment:
I think that we should keep `|| true` here because of:

* older clusters that might not have `versionlock` installed
* if some package isn't version-locked (I'm not sure is `versionlock` failing in that case -- IIRC, it doesn't, but we should double-check)

Reply:
* the `yum-plugin-versionlock` is already installed at this point so it's always present.
* `yum versionlock delete` is very liberal and if you pass it even non-existing packages it wouldn’t complain (e.g. giving you non-zero exist status)